include/types/wlr_buffer.h

```diff
@@ -0,0 +1,9 @@
+#include "wlr/types/wlr_buffer.h"
```

Comment:
This file needs include guards (`#ifndef TYPES_WLR_BUFFER_H`).

Reply:
Oh I took example from `types/wlr_keyboard.h` and it seems to be the only one that doesnt have an include guard.

Reply:
Oh, oops, that's an oversight. Can you send a patch to fix `wlr_keyboard.h`?

Reply:
Sure!